### PR TITLE
Add security level for parameters

### PR DESCRIPTION
**Pull request by pvoid**
It allows to exclude parameter value from exception message or include
some common info instead actual value

### CODE REVIEW

File: actors-library/src/main/java/com/truecaller/androidactors/MessageBase.java [hunk 1]
```diff
@@ -42,4 +43,33 @@ protected MessageBase(@NonNull ActorInvokeException exception) {
...
+    protected static String logParam(Object parameter, int level) {
+        if (level == SecureParameter.LEVEL_NO_INFO) {
+            return "<value>";
+        }
+
+        if (parameter == null) {
+            return "null";
+        }
+
+        if (parameter instanceof String) {
```
Comment (meandnano) [inline actors-library/src/main/java/com/truecaller/androidactors/MessageBase.java]: Maybe should check for `CharSequence` instead of `String`?

**Comment by opayen** [comment]
Code-wise it looks good to me, I'm just wondering about 2 things:
- unit tests
- update README.md to mention this new functionality

File: actors-library/src/main/java/com/truecaller/androidactors/MessageBase.java (hunk 1 above)
Comment (pvoid) [reply actors-library/src/main/java/com/truecaller/androidactors/MessageBase.java]: Done

**Comment by pvoid** [comment]
I've added unit tests, but updating README file is hard for me. I can write base text but I'll need someone who will review grammar and syntax.

File: actors-library/src/main/java/com/truecaller/androidactors/SecureParameter.java [hunk 2]
```diff
@@ -28,5 +28,5 @@
     int LEVEL_NULL_OR_EMPTY_STRING = 1;
     int LEVEL_FULL_INFO = 2;
 
-    int value() default LEVEL_FULL_INFO;
+    int value() default LEVEL_NULL_OR_EMPTY_STRING;
```
Comment (opayen) [inline actors-library/src/main/java/com/truecaller/androidactors/SecureParameter.java]: If you change default value here, you should probably also change it in here: https://github.com/truecaller/android-actors-library/pull/15/commits/ad33c7189ae9158b92dc9236bc4788bcd6841ebe#diff-82a44d7932d2bb1c7309ac716c55765fR257

**Review by opayen — APPROVED** [review]

**Pull request merged**